### PR TITLE
INVGOV-10386 Fixes to swagger schemas to remove minor mismatches

### PR DESCRIPTION
**Pull request by luke-knight**
https://io1dev.atlassian.net/browse/INVGOV-10386

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -6497,12 +6594,14 @@ components:
             - Note
             - Done
             - News / Event
-            - Suggestion
+            - Idea
```
Comment (MireilleHelle) [inline api.swagger.yaml]: These would align to Akagera data, but not what is setup on Live. So we shouldn't make this change.

Comment (sam-ludlow) [reply api.swagger.yaml]: I put it back

File: api.swagger.yaml [hunk 2]
```diff
@@ -6150,6 +6271,25 @@ components:
...
+              - Redacted
+              - Ghostwriter
+              - Brand manager
+              - Blogger
+              - Ideas administrator
+              - OU administrator
+              - Location admin
+              - Content layout manager
+              - External user
+              - Invotra support user
```
Comment (ParryLuke) [inline api.swagger.yaml]: Remove 'Invotra support user' and 'Services API user'/'Ghostwriter' as customers should not be able to set these.

Comment (sam-ludlow) [reply api.swagger.yaml]: Removed the 3 roles

**Pull request merged**